app/views/layouts/application.html.erb

```diff
@@ -5,9 +5,9 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <%= canonical_tag %>
-    <%= tag :meta, property: 'viewport', content: 'width=device-width, initial-scale=1' %>
-    <%= tag :meta, property: 'og:image', content: image_url('govuk-opengraph-image.png') %>
-    <%= tag :meta, property: 'theme-color', content: '#0b0c0c' %>
+    <%= tag :meta, name: 'viewport', content: 'width=device-width, initial-scale=1' %>
+    <%= tag :meta, name: 'og:image', content: image_url('govuk-opengraph-image.png') %>
```

Comment:
I think this one should still be `property`

Reply:
Maybe, it was the viewport one which was tripping up responsiveness, I'll check and revise the PR

Reply:
@dankmitchell  good catch, I've checked the syntax on the other two and those were correct. PR now revised